src/test/java/LoginTests.java

```diff
@@ -13,10 +13,30 @@ public static void LoginEmptyEmailPasswordTest () {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://testpro.io/";
+        String url = "https://bbb.testpro.io/";
         driver.get(url);
         Assert.assertEquals(driver.getCurrentUrl(), url);
         driver.quit();
         //changes from Natalies Branch
     }
+
+    @Test
+    public static void LoginValidEmailPasswordTest() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public static void loginEmptyEmailPasswordTest ()`